minikube_demo_setup.sh

```diff
@@ -44,6 +49,32 @@ function check_err() {
 	fi
 }
 
+# Prints the minimum system resources required to run the demo
+function print_min_resources() {
```

Comment:
Please use tabs for this PR

Reply:
@dinogun Thanks for the review I have made the changes to use Tabs. Thanks